operations/services.md

```diff
@@ -112,11 +113,18 @@ Positive end offsets of the daily time unit define the nightly service as depict
 Negative or zero end offsets of the daily time unit define the daily service as depicted on the picture below.
 ![](../.gitbook/assets/timeunits-connector-day.png)
 
-#### Time unit
+#### Time unit period
 
 * `Day`
```

Comment:
How about adding `Hour` to make it complete?

Reply:
We do not want to mention hour just yet, as there are still some open questions and we do not want to confuse partners.

Reply:
We recently had the discussion about enums with ellipsis or without and about the implications. 
This definitely feels like the hard requirement which should have all values listed. But on the other hand I understand the situation with rollout. So let's keep it this way for now.

Reply:
We could of course add it, but there will be no such service even for testing purposes probably. So then I am not 100% sure if it is ok to list it while it is not really supported. The thing with the monthly time unit is that it is not 100% supported either, but the pricing is in its end state so the partner can update those 2 endpoints and it will not change for the monthly unit. For hours, it is still not decided if you are going to update hour prices for every hour for every day. Maybe you will be updating just specific hours generally for all days the same or in some different batches, etc... That is the reason why we do not want to add it right now, even just for the pricing, because we cannot guarantee it will work in the same way at the moment,